chef/cookbooks/provisioner/templates/default/unattended.xml.erb

```diff
@@ -88,68 +88,72 @@
                     <Order>1</Order>
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
+                    <CommandLine>netsh advfirewall firewall set rule name="File and Printer Sharing (SMB-In)" new enable=yes</CommandLine>
+		    <Order>2</Order>
```

Comment:
It doesn't matter, but i grouped the netsh commands together. I could have added this as last in the Order.